test/types/date.test.ts

```diff
@@ -25,7 +27,10 @@ async function isInputsValid(month: string, day: string, year: string) {
     onSave = jest.fn(),
     props = { fieldSets, onSave };
 
-  const tester = await new Tester(FormCard, { props }).mount();
+  const tester = new Tester(FormCard, { props });
+  await act(async () => {
+    await tester.mount();
+  });
   await tester.changeInput(`input[id="${fieldConfig.field}.month"]`, month);
   await tester.changeInput(`input[id="${fieldConfig.field}.day"]`, day);
   await tester.changeInput(`input[id="${fieldConfig.field}.year"]`, year);
```

Comment:
Basically anything that isn't a precondition (like defining variables/factory instances or `onBeforeMount` setup) or assertion, and especially anything that interacts with the virtual DOM (even more especially for async interactions), should be wrapped in `act`.

So, all of the input interaction should also be wrapped, even though i'm pretty sure those awaits are doing nothing. I'd suggest that you fix the other tests in this file as well to future-proof, but all good if this is urgent. wrapping lines  34-38 would be sufficient

Reply:
relevant docs:
- https://reactjs.org/docs/testing-recipes.html#act
- http://wiki.c2.com/?ArrangeActAssert

Reply:
@rpalermodrums Yeah that makes sense. I left the rest of the actions as is because Robert updated our tester package to wrap everything that need to be wrapped in an `act` block like [`changeInput`](https://github.com/mighty-justice/tester/blob/master/src/tester.tsx#L114). though, it hasn't worked perfectly with `mount` and nested `act` blocks are okay so I updated the file to include all the items just in case